deploy/base/deployment.yaml

```diff
@@ -97,6 +104,13 @@ spec:
         - name: mixer
           image: gcr.io/datcom-ci/datacommons-mixer:latest
           imagePullPolicy: Always
+          resources:
```

Comment:
What fraction of the total node mem-size does this represent?

Reply:
Ok. Sg